txnpool/proc/txnpool_server.go

```diff
@@ -172,6 +170,7 @@ func (s *TXPoolServer) init(num uint8, preExec bool) {
 	}
 
 	s.gasPrice = getGasPriceConfig()
+	log.Infof("tx pool: initial enforced gas price %d", s.gasPrice)
```

Comment:
I think this message be " The current local gasprice is :xxxx " would be better.

Reply:
Updated